src/ui/server/api/keystone/countries/index.ts

```diff
@@ -23,11 +22,9 @@ const countries = {
         return response.data.countries;
       }
 
-      if (response instanceof ApolloError) {
-        throw new Error('Getting countries');
-      }
-    } catch {
-      throw new Error('Getting countries');
+      throw new Error(`Getting countries ${response}`);
+    } catch (err) {
+      throw new Error(`Getting countries ${err}`);
```

Comment:
Will these caught errors be eventually displayed by a calling stack function?

Reply:
Thanks @abhi-markan good shout - I just tried it out, doesn't work as intended unfortunately. The `err` is not logged out with `throw new Error( ...)` I propose changing them all to the following:

```js
} catch (err) {
  console.error(err);
  throw new Error('Creating exporter account');
}
```

So it will log out like this in the event of a failure

```js
Creating exporter account
Error: Creating exporter account Error: Network error: request to http://localhost:3000/api/graphql failed, reason: connect ECONNREFUSED ::1:3000
    at Object.create (/Users/tonyb/_projects/ukef/exip/src/ui/server/api/keystone/account/index.ts:28:13)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async Object.account (/Users/tonyb/_projects/ukef/exip/src/ui/server/controllers/insurance/account/create/your-details/save-data/index.ts:16:26)
    at async post (/Users/tonyb/_projects/ukef/exip/src/ui/server/controllers/insurance/account/create/your-details/index.ts:93:26)
Error creating account {
  err: Error: Creating exporter account
      at Object.account (/Users/tonyb/_projects/ukef/exip/src/ui/server/controllers/insurance/account/create/your-details/save-data/index.ts:20:11)
      at processTicksAndRejections (node:internal/process/task_queues:95:5)
      at async post (/Users/tonyb/_projects/ukef/exip/src/ui/server/controllers/insurance/account/create/your-details/index.ts:93:26)
}
```

Reply:
Updated this @abhi-markan so that we always `console.error` as well as throwing an error.

Reply:
Well done, exactly the same is what I am performing in DTFS.

Reply:
What are the chances 😊